runtime/bcutil/cfreader.c

```diff
@@ -2378,6 +2375,7 @@ j9bcutil_readClassFileBytes(J9PortLibrary *portLib,
 	/* protect the callers with default values */
 	if ((flags & BCT_MajorClassFileVersionMask) == 0) {
```

Comment:
I don't think this check is correct. It appears the MajorClassFileVersion portion of the flags is initialized further down in this function. It can be moved higher, but the classfile->majorVersion needs to be read in first. We shouldn't be initializing a default of Java 9.

Reply:
Just pushed a change to address this.